packages/flame_jenny/jenny/README.md

```diff
@@ -7,5 +7,8 @@ dialogue to users within a game or an app.
 
 The `jenny` library mimics the features and functionality of the [Yarn Spinner] library for Unity.
 
+See Jenny's [official documentation site] for more information.
```

Comment:
Maybe just keep to changes to this file and repurpose the PR

Reply:
Ok